setup.py

```diff
@@ -17,8 +17,16 @@
     entry_points={
         "lexibank.dataset": [
             "kpaamcam=lexibank_kpaamcam:Dataset"
-        ]
+        ],
+        'cldfbench.commands': [
+            'lowerfungom=kpaamcamcommands',
+        ],
```

Comment:
This makes the `lowerfungom.analyze` sub-command known to `cldfbench` - after `pip install -e .` is run to "install" the dataset.